### PR TITLE
Tools to check IPv4 or IPv6 input

### PR DESCRIPTION
**Pull request by Kashaf-Sajid-Yaseen**
Tools to check the type of input (IPv4 & IPv6) entered by the user.

### CODE REVIEW

**Comment by awaismslm** [comment]
@UmanShahzad Please check.

**Review by UmanShahzad — COMMENTED** [review]
Do fix issues commonly found in the other PRs that also appear here, then do ping me again @Kashaf-Sajid-Yaseen @awaismslm

**Comment by Kashaf-Sajid-Yaseen** [comment]
@UmanShahzad Please check.

File: ipinfo/cmd_tool.go [hunk 1]
```diff
@@ -62,6 +66,10 @@ func cmdTool() error {
 	switch {
 	case cmd == "aggregate":
 		err = cmdToolAggregate()
+	case cmd == "is_v4":
+		err = cmdToolIs_v4()
+	case cmd == "is_v6":
+		err = cmdToolIs_v6()
```
Comment (UmanShahzad) [inline ipinfo/cmd_tool.go:72]: Func name should be `cmdToolIsV6` (Similar for v4), even though the actual subcmd has an underscore.

File: ipinfo/cmd_tool_is_v6.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	fmt.Printf(
+		`Usage: %s tool is_v6 [<opts>] <cidr | ip | ip-range | filepath>
+
+Description:
+  Checks if the input is an IPv6 address.
+  Inputs can be IPs, IP ranges, CIDRs, or filepath to a file
+
+Examples:
+  # Check CIDR.
+  $ %[1]s tool is_v6 1.1.1.0/30
```
Comment (UmanShahzad) [inline ipinfo/cmd_tool_is_v6.go:31]: Can we give a real v6 example

**Review by UmanShahzad — COMMENTED** [review]
Few things:

- Ensure we dont use underscores in the name of Golang functions. In the subcommand name itself it's fine but not in golang code.
- For printing, we should just print `<ip>,true` or `<ip>,false` for now instead of the full sentences of `<ip> is na IPv4 input`
- Can we develop helper functions to check if the IP or IPNet (cidr) is v4/v6? Too many inlined conditionals are being used which makes it harder to read and reuse.

**Comment by Kashaf-Sajid-Yaseen** [comment]
@UmanShahzad Please check.

**Review by UmanShahzad — APPROVED** [review]

**Pull request merged**